frontend/src/APIClients/AdminAPIClient.ts

```diff
@@ -0,0 +1,20 @@
+import { BEARER_TOKEN } from "../constants/AuthConstants";
+import { UpdateWaiverRequest } from "../types/AdminTypes";
+import baseAPIClient from "./BaseAPIClient";
+
+const updateWaiver = async (
+  req: UpdateWaiverRequest,
```

Comment:
is it standard practise to call this req? If not, can we rename this to `updateWaiverData`